homeassistant/components/sensibo/climate.py

```diff
@@ -148,6 +149,9 @@ def __init__(self, client, data, units):
         """
         self._client = client
         self._id = data["id"]
+        self._productmodel = data["productModel"]
+        self._firmware = data["firmwareVersion"]
+        self._firmwaretype = data["firmwareType"]
```

Comment:
Create the deviceinfo directly here if possible:
```suggestion
        self._attr_device_info = DeviceInfo(
            model = data["productModel"],
            sw_version = data["firmwareVersion"],
            hw_version = data["firmwareType"],
            ...
        )
```

Reply:
I was thinking this as a "quick fix" and then rewrite the whole platform into the attr_* format.
But I will fix it first for this one then.